task3-4/task3-4/ViewsInStackViewController.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  StackViewController.swift
+//  task3-4
+//
+//  Created by Григоренко Александр Игоревич on 16.10.2022.
+//
+
+import UIKit
+
+/// Экран сверстанный с помощью стеквью
+final class ViewsInStackViewController: UIViewController {
+
+    // MARK: - visual components
+
+    private var bigView = UIView()
+    private var redView = UIView()
+    private var yellowView = UIView()
+    private var greenView = UIView()
+    private var stackView = UIStackView()
+
+    // MARK: - life cycle
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        addBigView()
+    }
+
+    // MARK: - private funcs
+
+    private func addBigView() {
+        view.backgroundColor = #colorLiteral(red: 0.9764705896, green: 0.850980401, blue: 0.5490196347, alpha: 1)
+        view.addSubview(bigView)
+        bigView.addSubview(stackView)
+
+        bigView.backgroundColor = #colorLiteral(red: 0, green: 0, blue: 0, alpha: 1)
```

Comment:
```suggestion
        bigView.backgroundColor = #colorLiteral(red: 0, green: 0, blue: 0, alpha: 1)
```
Цвета в Assets в следующий раз!